src/main/java/com/dataloom/authorization/requests/PropertyTypeInEntitySetAclRequestWithRequestingUser.java

```diff
@@ -0,0 +1,61 @@
+package com.dataloom.authorization.requests;
+
+import com.dataloom.data.SerializationConstants;
+import com.dataloom.edm.requests.PropertyTypeInEntitySetAclRequest;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class PropertyTypeInEntitySetAclRequestWithRequestingUser {
```

Comment:
These wrapper classes are getting out of hand. We need to make these objects more sane.